### PR TITLE
Adding a blog post "It's not a wonder, women"

### PR DESCRIPTION
**Pull request by OritGeron**

### CODE REVIEW

File: src/_posts/2017-03-07-its-not-a-wonder-women.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: It's not a wonder, women
+tags:
+  - Office Culture
+permalink: "/content/its-not-a-wonder-women/"
+layout: post
+image:
+author: OritiMG
+published: true
+description:
```
Comment (RachelBaram) [inline src/_posts/2017-03-07-its-not-a-wonder-women.md]: @OritGeron Please add a description - it should be less than 160 chars (target = 150-160) and be a good summary of of the content with the appropriate keywords.

**Comment by RachelBaram** [comment]
@OritGeron Please give it a final pass, write a description, and upload the pictures you would like.  Consider taking out the last sentence on line 24 -  "Imagine my surprise to find out, despite my stereotyped feelings, that all of my presumptions were faded away. There’s no drama, no hidden intentions, and no judgement. The blunt-in-your-face attitude is there as well - who knew?!?"  Once you are completely done, take off WIP, and I will do a final proofread. Kol Hakavod!

**Comment by OritGeron** [comment]
oneoftheguys

img_0372

everywomanisawonder

**Comment by OritGeron** [comment]
@RachelBaram darling please review description. I removed the rest of line 24.
PC sucks ;0)

File: src/_posts/2017-03-07-its-not-a-wonder-women.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+title: It's not a wonder, women
```
Comment (IshaDakota) [inline src/_posts/2017-03-07-its-not-a-wonder-women.md]: Title Case

**Comment by RachelBaram** [comment]
@OritGeron Looks good my dear awesome woman co-worker.

File: src/_posts/2017-03-07-its-not-a-wonder-women.md [hunk 3]
```diff
@@ -16,10 +16,10 @@ I must admit that I am guilty of believing this sentiment. I felt it growing up.
...
+Even in this discomfort, I understood that Gizra is not typical for its views about diversity. I felt grateful and kept thinking to myself, how is it even possible that more people don’t know this about Gizra? How often do you hear of a company that wants diversity, wants women because they know it will make their company even better!? A company that knows that you have a **family life** outside of work and it’s not a dirty word.
+A company that understands that if you want to have a **career**, your **children** are not an obstacle. A recent example - my son had to stay home from school this week so I took him to the office. While I was caught up with my work, I realized that it was too quiet and immediatly went out on a persuit, because everybody knows that *kids + quite = truoble*!. To my relief, I found him happily eating lunch with Amitai and the rest of the developers.
```
Comment (OritGeron) [inline src/_posts/2017-03-07-its-not-a-wonder-women.md]: @IshaDakota Made a few changes, please review

File: src/_posts/2017-03-07-its-not-a-wonder-women.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
+---
+title: It's Not a Wonder, Women.
+tags:
+  - Office Culture
```
Comment (OritGeron) [inline src/_posts/2017-03-07-its-not-a-wonder-women.md]: @RachelBaram 
office culture? is there anything else? I just copied it from another blog post..

File: src/_posts/2017-03-07-its-not-a-wonder-women.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+I really look forward to those days that all of the women are in the office - to share failures, wins, worries, and celebrations. We are friends. We are professional co-workers. We have each other’s back. We support and encourage each other. We are happy for the other’s successes. We help move the company forward, and there’s nothing we can't accomplish. These ladies makes me a better person, a better mother, a better woman!
+
+In Gizra, we always ask how we can do it better. This is doing it better, and we still have a way to go.  We understand very well the importance of **“fit”** with our clients and this is also true for potential employees. So yes. **Sometimes you just need to be in a place so special that it fits**. So, if you are looking for a fit and this sounds right to you, You can apply. And if you are a women, you are (more than) welcome to apply!
```
Comment (ordavidil) [inline src/_posts/2017-03-07-its-not-a-wonder-women.md]: @IshaDakota 
>  And if you are a women

shouldn't it be `a woman` ?
or just `women` ?

Comment (RachelBaram) [inline src/_posts/2017-03-07-its-not-a-wonder-women.md]: Yes.

File: src/_posts/2017-03-07-its-not-a-wonder-women.md (hunk 4 above)
Comment (RachelBaram) [inline src/_posts/2017-03-07-its-not-a-wonder-women.md]: I think that is fine.

**Comment by IshaDakota** [comment]
image

**Comment by IshaDakota** [comment]
gizra_s_blog

**Comment by ordavidil** [comment]
Seems to have some leftovers under the `Its-not-a-wonder-women` directory (pay attention to the uppercased `I`).

**Comment by IshaDakota** [comment]
> Seems to have some leftovers

That should do it.

**Comment by OritGeron** [comment]
@RachelBaram @amitaibu 
 Are we good to go? Today is the day (yay!)

**Comment by RachelBaram** [comment]
From a content perspective... looks good! It's up to @amitaibu to release
(or assign someone to do it).  ;)

On Mar 8, 2017 09:35, "Orit" <notifications@github.com> wrote:

@RachelBaram <https://github.com/RachelBaram> @amitaibu
<https://github.com/amitaibu>
Are we good to go? Today is the day (yay!)

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/Gizra/gizra.com/pull/581#issuecomment-284968855>, or mute
the thread

**Comment by nader77** [comment]
@amitaibu I can do it if you are not available today :)

**Comment by RachelBaram** [comment]
@nader77 If you see that from a technical perspective its all good, please go ahead! Thanks! cc: @amitaibu

**Comment by nader77** [comment]
Alright, reviewing.

**Comment by nader77** [comment]
it_s_not_a_wonder__women_

gizra_s_blog

**Comment by nader77** [comment]
Closing this PR and opening a new one because there's a lot of irrelevant files.

**Comment by nader77** [comment]
Closed in favor of #584

**Pull request closed without merging**